The following features are disabled:

* monitor-mode sniffing of wireless interfaces on Linux, via libnl
* building shared libraries
* Bluetooth support
* canusb support
* CAN support
* dbus capture support

diff --git a/libpcap/configure.in b/libpcap/configure.in
index 5386062..8ceddd7 100644
--- a/libpcap/configure.in
+++ b/libpcap/configure.in
@@ -441,10 +441,10 @@ linux)
 	# Do we have libnl?
 	#
 	AC_ARG_WITH(libnl,
-	AC_HELP_STRING([--without-libnl],[disable libnl support @<:@default=yes, on Linux, if present@:>@]),
+	AC_HELP_STRING([--without-libnl],[disable libnl support @<:@default=disabled@:>@]),
 		with_libnl=$withval,,)
 
-	if test x$with_libnl != xno ; then
+	if test x$with_libnl = xyes ; then
 		have_any_nl="no"
 
 		#
@@ -1285,8 +1285,8 @@ solaris*)
 esac
 
 AC_ARG_ENABLE(shared,
-AC_HELP_STRING([--enable-shared],[build shared libraries @<:@default=yes, if support available@:>@]))
-test "x$enable_shared" = "xno" && DYEXT="none"
+AC_HELP_STRING([--enable-shared],[build shared libraries @<:@default=no@:>@]))
+test "x$enable_shared" != "xyes" && DYEXT="none"
 
 AC_PROG_RANLIB
 AC_CHECK_TOOL([AR], [ar])
@@ -1434,9 +1434,9 @@ AC_SUBST(PCAP_SUPPORT_NETFILTER)
 AC_SUBST(NETFILTER_SRC)
 
 AC_ARG_ENABLE([bluetooth],
-[AC_HELP_STRING([--enable-bluetooth],[enable Bluetooth support @<:@default=yes, if support available@:>@])],
+[AC_HELP_STRING([--enable-bluetooth],[enable Bluetooth support @<:@default=no@:>@])],
     [],
-    [enable_bluetooth=yes])
+    [enable_bluetooth=no])
 
 if test "x$enable_bluetooth" != "xno" ; then
 	dnl check for Bluetooth sniffing support
@@ -1480,9 +1480,9 @@ if test "x$enable_bluetooth" != "xno" ; then
 fi
 
 AC_ARG_ENABLE([canusb],
-[AC_HELP_STRING([--enable-canusb],[enable canusb support @<:@default=yes, if support available@:>@])],
+[AC_HELP_STRING([--enable-canusb],[enable canusb support @<:@default=no@:>@])],
     [],
-    [enable_canusb=yes])
+    [enable_canusb=no])
 
 if test "x$enable_canusb" != "xno" ; then
 	dnl check for canusb support
@@ -1507,9 +1507,9 @@ if test "x$enable_canusb" != "xno" ; then
 fi
 
 AC_ARG_ENABLE([can],
-[AC_HELP_STRING([--enable-can],[enable CAN support @<:@default=yes, if support available@:>@])],
+[AC_HELP_STRING([--enable-can],[enable CAN support @<:@default=no@:>@])],
     [],
-    [enable_can=yes])
+    [enable_can=no])
 
 if test "x$enable_can" != "xno" ; then
 	dnl check for CAN sniffing support
@@ -1532,9 +1532,9 @@ if test "x$enable_can" != "xno" ; then
 fi
 
 AC_ARG_ENABLE([dbus],
-[AC_HELP_STRING([--enable-dbus],[enable D-Bus capture support @<:@default=yes, if support available@:>@])],
+[AC_HELP_STRING([--enable-dbus],[enable D-Bus capture support @<:@default=no@:>@])],
     [],
-    [enable_dbus=ifavailable])
+    [enable_dbus=no])
 
 if test "x$enable_dbus" != "xno"; then
 	AC_CHECK_PROG([PKGCONFIG], [pkg-config], [pkg-config], [no])
